src/main/java/net/sf/jabref/logic/undo/AddUndoEvent.java

```diff
@@ -0,0 +1,18 @@
+package net.sf.jabref.logic.undo;
+
+
+public class AddUndoEvent {
+
+    public final boolean canUndo;
```

Comment:
Hide these fields behind getters (no setters). And rename `undoText` to `undoDescription`